index.html

```diff
@@ -1,14 +1,254 @@
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style/style.css">
+  <link rel="stylesheet" href="style/home.css">
+  <link rel="stylesheet" href="style/about.css">
+  <title>Velzck Capstone Project</title>
 </head>
-<body>
-    <script src="script.js"></script>
+
+<body ontouchstart>
+  <!-- HEADER -->
+  <header id="top">
+    <div id="accessibilityCont">
+      <ul id="accessibility">
+        <li class="accessibilityOption"><a href="https://www.facebook.com/" target="_blank"
+            rel="noopener noreferrer"><img class="accessibilityIcon" src="style/icons/facebook.svg"
+              alt="facebook icon"></a></li>
+        <li class="accessibilityOption"><a href="https://twitter.com/velzck" target="_blank"
+            rel="noopener noreferrer"><img class="accessibilityIcon" src="style/icons/twitter.svg"
+              alt="twitter icon"></a></li>
+        <div id="languageWrapper">
+
+          <ul id="languageCont">
+            <li class="language">english</li>
+            <li class="language">spanish</li>
+            <li class="language">italian</li>
+            <li class="language">russian</li>
+            </li>
+          </ul>
+
+        </div>
+        <li class="accessibilityOption" id="register">register</li>
+        <li class="accessibilityOption" id="log-in">log in</li>
+    </div>
+    <div id="menuCont">
+      <div id="menu">
+        <button type="button" id="menuBtn" class="menuAction" aria-label="menu button"><img
+            src="style/icons/hamb-bars.svg" alt="menu icon"></button>
+        <div class="logoCont">
+          <a href="#top">
+            <img id="menuLogo" src="style/logo/header-logo2.webp" alt="logo">
+          </a>
+        </div>
+        <ul id="menuBar">
+          <li class="menuItem"><a class="menuLink menuAction homeButton" href="#">home</a></li>
+          <li class="menuItem homeSection"><a class="menuLink menuAction" href="#mainProgramCont">program</a></li>
+          <li class="menuItem homeSection"><a class="menuLink menuAction" href="#speakersCont">featured</a></li>
+          <li class="menuItem"><a class="menuLink menuAction aboutButton" href="#">about</a></li>
+          <li class="menuItem aboutSection"><a class="menuLink menuAction" href="#logoCont">logo</a></li>
+          <li class="menuItem"><a id="campaignBtn" class="menuLink menuAction" href="#festival">A&D Festival</a></li>
+        </ul>
+      </div>
+    </div>
+
+  </header>
+
+  <!--   HOMEPAGE -->
+  <section id="homePage">
+
+    <!-- HERO -->
+    <section id="heroCont">
+
+      <div id="hero">
+        <h3 class="heroTitle">"hello designers of the world!"</h3>
+        <h1 class="heroTitle">architecture and designers festival <br>2023</h1>
+        <p id="heroDescription">Welcome to the intrnational <span id="heroSpan">Architecture and Design festival
```

Comment:
- As requested by the previous reviewer [here](https://github.com/VelzckC0D3/capstone_project/pull/1/files#r1140931268)  kindly make sure h1 always come before h2,h3,... in order to meet the semantic HTML syntax.

__Note: only comments starting with [OPTIONAL] are optional, all others are mandatory for the approval of your project__